pkg/controller/statusmanager/status_manager_test.go

```diff
@@ -253,6 +253,59 @@ func TestStatusManager_set(t *testing.T) {
 	}
 }
 
+func TestStatusManager_set_OpenShiftSDN(t *testing.T) {
+	client := fake.NewFakeClient()
+	status := New(client, "testing", "")
+
+	// make the network.operator object
+	log.Print("Creating Network Operator Config")
+	no := &operv1.Network{
+		ObjectMeta: metav1.ObjectMeta{Name: names.OPERATOR_CONFIG},
+		Spec: operv1.NetworkSpec{
+			DefaultNetwork: operv1.DefaultNetworkDefinition{
+				Type: operv1.NetworkTypeOpenShiftSDN,
+			},
+		},
+	}
+
+	set(t, client, no)
+
+	condNoProgress := operv1.OperatorCondition{
+		Type:   operv1.OperatorStatusTypeProgressing,
+		Status: operv1.ConditionFalse,
+	}
+	condAvailable := operv1.OperatorCondition{
+		Type:   operv1.OperatorStatusTypeAvailable,
+		Status: operv1.ConditionTrue,
+	}
+
+	// Check if OpenShiftSDN will make operator non-upgradable
```

Comment:
NIT: This is redundant, network type is already set to sdn here: https://github.com/openshift/cluster-network-operator/pull/2330/files#diff-faa3843b60b4c6b2bdc956d8fef07559b04afb279aa13b08d607d0bac4831550R266